src/AppBundle/Admin/UserAdmin.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace AppBundle\Admin;
+
+use AppBundle\Entity\User;
+use Sonata\AdminBundle\Admin\Admin;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\DoctrineORMAdminBundle\Datagrid\ProxyQuery;
+
+class UserAdmin extends Admin
+{
+    protected $baseRouteName = 'AppBundle\Entity\User';
+    protected $baseRoutePattern = 'User';
+    protected $datagridValues = [
+        '_sort_order' => 'DESC',
+        '_sort_by' => 'id',
+    ];
+
+    /**
+     * @param \Sonata\AdminBundle\Form\FormMapper $formMapper
+     */
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        $formMapper
+            ->with('User', ['class' => 'col-lg-12'])
+            ->add('firstName')
+            ->add('lastName')
+             ->add('email')
+
+            ->add('facebookId')
+            ->add('apiKey', 'text', ['attr' => ['readonly' => true]])
```

Comment:
remove this. We will have "impersonate" functionality, but apiKey should be secured